README.md

```diff
@@ -162,3 +162,34 @@ you'll want to set one or the other, not both.
   with:
     pylance-version: latest-release
 ```
+
+## Keeping Pyright in sync with your project
+
+If you use Pyright in your project out of the CI, chances are that the Pyright
+is already specified somewhere. We'll try to share in this section recipes for
+tying the version of Pyright used in the CI to the version used in your project.
+Those are generic recipes that you may need to adapt to your project's needs.
+
+### `pre-commit`
+
+If you're using `https://github.com/RobertCraigie/pyright-python` or any other
+pre-commit hook where the version of Pyright is the revision of the repo (with a
+leading `v` character), you may use the following snippet:
+
+```yaml
+- name: Extract pyright version from pre-commit
+  id: pre-commit-pyright-version
+  run: |
+    yq '.repos
```

Comment:
Is `yq` just `jq` but for yaml? Does it need the `-r` flag to get rid of the quotes? I know `jq` does.

Reply:
> Is `yq` just `jq` but for yaml?

[Yep](https://mikefarah.gitbook.io/yq/), well, it started that way, but now `yq` handles many different formats including toml. It's a bit less powerful than jq, but a bit more versatile.

> Does it need the -r flag to get rid of the quotes? I know jq does.

`-r` does that indeed but default to true for yaml, so we don't need it here
```
  -r, --unwrapScalar                  unwrap scalar, print the value with no quotes, colors or comments. Defaults to true for yaml (default true)
```